OCC.Service.Webhost.Tests/Session/GetSessionTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Linq;
+using System.Linq.Expressions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OCC.Data;
+using OCC.Service.Webhost.Tests.Helpers;
+
+namespace OCC.Service.Webhost.Tests.Session
+{
+    [TestClass]
+    public class GetSessionTests
+    {
+        private Data.Session _expectedSession;
+        private Event _expectedEvent;
+        private Person _expectedSpeaker;
+
+        private Services.Session _actualSession;
+
+        [TestInitialize]
```

Comment:
I used to keep these scenarios all in the same test, but one of the downsides is when the first assert fails you receive no feedback on the other asserts. Often you will find yourself fixing one issue just to re-run the tests and fall to the next failure. This gives you a test for all of the properties and one property/mapping issue will not prevent feedback for the other values.

There definitely need to be more tests implemented around everything, but I wanted to get a couple of tests pushed in to show the full working implementation with the DI container and Effort.